initial commit

jbuccola opened the pull request.

johnny-buccola committed:
initial commit

johnny-buccola committed:
adding old homework files

jbuccola commented:
adding past-due homework

Borgaard commented on Johnny-jbuccola/homework_11/app.js:
```diff
@@ -0,0 +1,54 @@
+      });
+    });
+  } else {
+    $('.images').append("Sorry your browser sucks");
```
Classy, haha.

johnny-buccola committed:
adding homework 13 - stocks!

Borgaard commented on Johnny-jbuccola/homework_11/index.html:
```diff
@@ -0,0 +1,37 @@
+      <h3>Discover how beautiful your surroundings can be</h3>
+      <button class="btn btn-default">
+        <a href="#" id="login">Login to 500px</a>
+      </button>
```
Consideration for later when we work with auth functionality: 
When a user is logged in, what happens to this `.login` id? Would you use templates to hide/show a login button depending on whether the user is authenticated? Would you use vanilla JS/jQuery to build the page upon determining authentication status?

johnny-buccola committed:
1adding 14- 'this' and 'module' homeworks

honestly, I'm not totally sure I did this correctly... I was a little unclear on the requirements

johnny-buccola committed:
Feedr Project Commit

...only thing that doesn't quite work... THE STUPID PACMAN WON'T GO AWAY

jbuccola commented:
Adding feedr project
